challenges/objects-arrays.js

```diff
@@ -43,26 +67,32 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
-
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+const universities = graduates.map(function(element) {
+	return element.university;
+});
 
-/* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
+console.log(universities);
 
+/* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
-
 Log the result of your new array. */
-const contactInfo = [];
+const contactInfo = graduates.map(function(element) {
+	let firstNameAndEmail = element.first_name + " " + element.email;
+	return firstNameAndEmail;
+});
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
-const uni = [];
+const uni = graduates.filter(function(element) {
+	return element.university.substring(0, 3) === 'Uni'; 
+});
```

Comment:
The same issue pertains to answers 2 and 3 also. You used array methods and were instructed not to.